qa/os/src/test/java/org/elasticsearch/packaging/test/PackagingTestCase.java

```diff
@@ -131,4 +158,79 @@ protected Shell newShell() throws Exception {
         }
         return sh;
     }
+
+    public Shell.Result startElasticsearch() throws Exception {
+        if (distribution.isPackage()) {
+            return Packages.startElasticsearch(sh);
+        } else {
+            assertThat(distribution.isArchive(), equalTo(true));
+            return Archives.startElasticsearch(installation, sh);
+        }
+    }
+
+    public Shell.Result startElasticsearchStandardInputPassword(String password) {
+        assertThat("Only archives support passwords on standard input", distribution().isArchive(), equalTo(true));
+        return Archives.startElasticsearch(installation, sh, password);
+    }
+
+    public Shell.Result startElasticsearchTtyPassword(String password) throws Exception {
+        assertThat("Only archives support passwords on TTY", distribution().isArchive(), equalTo(true));
```

Comment:
Here and elsewhere, `equalTo(true)` is unnecessarily verbose. `assertTrue` should work, I think.